apps/framework-cli/src/cli/routines.rs

```diff
@@ -344,9 +344,12 @@ async fn process_pubsub_message(
 fn start_leadership_lock_task(redis_client: Arc<Mutex<RedisClient>>, project: Arc<Project>) {
     tokio::spawn(async move {
         let mut interval = interval(Duration::from_secs(LEADERSHIP_LOCK_RENEWAL_INTERVAL)); // Adjust the interval as needed
+
+        let cron_registry = CronRegistry::new().await.unwrap();
```

Comment:
cool this fixes it.

Reply:
it's amusing how after some changes i managed to make the same silly mistake lol